.ci/upload.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+rm -rf dist
+mkdir dist/
+
+export STORAGE_ACCOUNT_NAME=platopslackhelpbotai
+export CONTAINER_NAME=the-hmcts-way
+
+function extractMain () {
+    local path=$1
+
+    DIRECTORY=$(dirname "$path")
+    mkdir -p dist/$DIRECTORY
```

Comment:
I need to install `htmlq` on the line below before this is ready, I forgot about that

https://github.com/mgdm/htmlq/releases/download/v0.4.0/htmlq-x86_64-linux.tar.gz